packages/gestalt/src/Icon/InternalIcon.tsx

```diff
@@ -0,0 +1,185 @@
+import classnames from 'classnames';
+import styles from '../Icon.css';
+import compactIconsClassic from '../icons/compact/index';
+import icons from '../icons/index';
+import compactIconsVR from '../icons-vr-theme/compact/index';
+import vrIcons from '../icons-vr-theme/index';
+import useInExperiment from '../useInExperiment';
+
+export type IconColor =
+  | 'default'
+  | 'disabled'
+  | 'subtle'
+  | 'success'
+  | 'error'
+  | 'warning'
+  | 'info'
+  | 'recommendation'
+  | 'inverse'
+  | 'shopping'
+  | 'brandPrimary'
+  | 'light'
+  | 'dark';
+
+type IconName = keyof typeof icons | keyof typeof compactIconsVR | keyof typeof compactIconsClassic;
```

Comment:
type IconName = keyof typeof icons | keyof typeof compactIconsVR | keyof typeof compactIconsClassic;

do we need : ```| keyof typeof compactIconsClassic;```?

Reply:
I guess all VR icons will contain classic right? So maybe you are correct that we don't need this